kentik_api_library/kentik_api/api_calls/api_call.py

```diff
@@ -1,6 +1,9 @@
 # Standard library imports
 from enum import Enum
 
+# ID that identifies resource in URL
+ResourceID = int
```

Comment:
Well, using ID throughout api_calls package was the first thought.
It would obviously work, although if you wanted to change your mind and represent ID as a different type, that could impact entire api_calls package.
Specifically, the value is low coupling as per GRASP principle.